rocketpy/motors/TankGeometry.py

```diff
@@ -2,7 +2,7 @@
 
 from rocketpy import Function
 from rocketpy.Function import PiecewiseFunction, funcify_method
-from functools import cached_property
+from functools import cached_property, cache
```

Comment:
This probably is causing problems with testing too


```suggestion
from functools import cache
try:
    from functools import cached_property
except ImportError:
    from .tools import cached_property
```